packages/crypto/package.json

```diff
@@ -1,15 +1,26 @@
 {
   "name": "@penumbra-zone/crypto-web",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "license": "MIT",
   "type": "module",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": "./dist/index.js"
+  },
   "scripts": {
+    "build": "pnpm clean && tsc && vite build",
+    "clean": "rm -rfv dist",
     "lint": "eslint \"**/*.ts*\"",
     "test": "vitest run"
   },
   "dependencies": {
     "bip39": "^3.1.0",
-    "crypto-js": "^4.2.0"
+    "crypto-js": "^4.2.0",
+    "@penumbra-zone/types": "workspace:*"
```

Comment:
Two reasons why `@penumbra-zone/types` was put back in the local directories:
- Cyclical dependencies (not detected when in parent package.json)
- Turbo's automatic detection of dependencies did not work without it